src/app.cc

```diff
@@ -50,34 +51,77 @@ void Init() {
   ShaderManager::CreateAndLink();
   ShaderManager::Bind();
 
+  // Camera initialization
+  Camera::Init();
+
+  // Temporary part, todo move this into a factory or something
+
   // Particle system initialization
```

Comment:
All of this initialization should be placed somewhere else. When a gui is available, just remove and select programmatically all the steps necessary to select a type of particle.